models/ipfs_networks.go

```diff
@@ -19,9 +19,12 @@ type HostedIPFSPrivateNetwork struct {
 	Name      string         `gorm:"unique;type:varchar(255)"`
 	APIURL    string         `gorm:"type:varchar(255)"`
 	SwarmKey  string         `gorm:"type:varchar(255)"`
-	Users     pq.StringArray `gorm:"type:text[]"` // these are the users to which this IPFS network connection applies to specified by eth address
+	Users     pq.StringArray `gorm:"type:text[]"`
 	Activated time.Time
```

Comment:
this is already determined by `Activated` - if time.Time is zero-value then the network is offline. I think maintaining 2 fields that say the same thing will be prone to breakage

Reply:
Derp

Reply:
I do think we might want to revisit this at some point - I'll keep thinking about it, and will come up with something following [TEM-155](https://rtradetechnologies.atlassian.net/browse/TEM-155)